README.md

```diff
@@ -163,18 +163,23 @@ app.use(plugin)
 `done` should be called only once, when your plugin is ready to go.  Additional
 calls to `done` are ignored.
 
-async/await is also supported:
+
+`use` returns a thenable wrapped instance on which `use` is called, to support a chainable API that can also be awaited.
+
+This way, async/await is also supported and `use` can be awaited instead of using `after`:
 
 ```js
-async function plugin (server, opts) {
-  await sleep(10)
+async function main () {
```

Comment:
Since the lead-in says "instead of using `after`:", I think it would be a good idea to show both examples. Something like:

> Using `after`:
> ```js
> // example
> ```
>
> Using the thenable:
> ```js
> // your current example
> ```

Reply:
good idea :+1: